source/scss/components/_messages.scss

```diff
@@ -0,0 +1,62 @@
+.fti-message {
+  color: $white;
+  padding: .875rem .9375rem;
+
+  &.info-message {
```

Comment:
We already have a molecule for info messages (`bg-info`). Maybe we should use that one instead of having a new class... What do you think?

Reply:
This is a CSS class meant only for setting a background color, isn't it?

Reply:
Exactly. But this `info-message` is doing the same thing

Reply:
I prefer to keep it, because in the future, the default configuration for the `.fti-message` class will have some rules that the Info Message will not. This is not visible now because I only added the Info Message on this ticket.
For example, all messages will be `position: fixed`, and only the Info Message will remain `static`. Therefore now we can only see the `background-color` difference.

Reply:
I see. Makes sense on this case to keep it.

Reply:
Right now it's good to use Bruno's suggestion. If something comes later, it will be part of a ticket later. This is only about the background color. We can use SCSS strengths in order to use both bg-info and the rules that you will add in the future.

Using @extend you can use the styles of a default style and add your own additional once:
```
.error {
  border: 1px #f00;
  background-color: #fdd;
}
.seriousError {
  @extend .error;
  border-width: 3px;
}
```

compiles to:
```
.error, .seriousError {
  border: 1px #f00;
  background-color: #fdd;
}

.seriousError {
  border-width: 3px;
}
```

your HTML:
```
<div class="error seriousError">
  Oh no! You've been hacked!
</div>
```